kong/pdk/log.lua

```diff
@@ -690,6 +694,33 @@ do
     return root
   end
 
+  -- create authenticated_entity and tls_info tables for both http and non-http serializing
+  -- @param ctx the ngx.ctx table
+  -- @param var the ngx.var table
+  -- @return authenticated_entity, tls_info
+  local function prepare_serialize(ctx, var)
```

Comment:
I think that `prepare_serialize` may not a good name.

Could we split this function to two small functions `get_authenticated_entity` and `get_tls_info`,

This may let the code more readable.

Reply:
> I think that `prepare_serialize` may be not a good name.
> 
> Could we split this function to two small functions `get_authenticated_entity` and `get_tls_info`,
> 
> This may let the code more readable.

updated